packages/main/src/MenuItem.js

```diff
@@ -66,7 +78,7 @@ const metadata = {
 		},
 
 		/**
-		 * Indicates if the any of the element siblings have icon.
+		 * Indicates if any of the element siblings have icon.
```

Comment:
Indicates whether any of the element siblings have an icon.

Reply:
Done